README.md

```diff
@@ -216,7 +269,7 @@ Injector injector = Guice.createInjector(new StorageModule(config));
 DistributedStorage storage = injector.getInstance(StorageService.class);
 ```
 
-**Note** : refers to the class constructor and the *createDistributedStorage()* method of [LocallyConfiguredCassandraFactory.java](https://github.com/scalar-labs/indetail/blob/master/Q%26A_application/backend/QA/src/main/java/com/example/qa/dao/LocallyConfiguredCassandraFactory.java) for more details
+**Note** : refers to the class constructor and the *createDistributedStorage()* method of [LocallyConfiguredStorageFactory.java](https://github.com/scalar-labs/indetail/blob/master/Q%26A_application/backend/QA/src/main/java/com/example/qa/dao/LocallyConfiguredStorageFactory.java) for more details
```

Comment:
https://github.com/scalar-labs/getting-started-with-scalardb/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R300

Is it true for Cosmos? (asking because I don't think so)

Reply:
Sorry, This document supports not only Cosmos DB but also Cassandra.

Reply:
so, is it true?

Reply:
> is it true? 

yes, It will support Cosmos DB